### ISSUE TITLE
Change post interaction buttons to <button> element for accessibility

### ISSUE THREAD

**Issue by carlydotwebsite** [pixelfed/pixelfed#756]
Right now they're marked up as h3 tags, which can't be tabbed to or clicked with keyboard navigation. If I recall correctly, the best practice for making this kind of functionality accessible is to use the button tag, which I think should be an easy-ish switch?
